src/NUnitFramework/tests/Constraints/UniqueItemsConstraintTests.cs

```diff
@@ -41,17 +41,17 @@ public void SetUp()
         static object[] FailureData = new object[] { new object[] { new int[] { 1, 3, 17, 3, 34 }, "< 1, 3, 17, 3, 34 >" } };
 
         [Test]
-        [TestCaseSource( "IgnoreCaseData" )]
-        public void HonorsIgnoreCase( IEnumerable actual )
+        [TestCaseSource("IgnoreCaseData")]
+        public void HonorsIgnoreCase(IEnumerable actual)
         {
-            Assert.That( new UniqueItemsConstraint().IgnoreCase.ApplyTo( actual ).IsSuccess, Is.False, "{0} should be unique ignoring case", actual );
+            Assert.That(new UniqueItemsConstraint().IgnoreCase.ApplyTo(actual).IsSuccess, Is.False, "{0} should be unique ignoring case", actual);
         }
 
         private static readonly object[] IgnoreCaseData =
         {
-            new object[] {new SimpleObjectCollection("x", "y", "z", "Z")},
-            new object[] {new[] {'A', 'B', 'C', 'c'}},
-            new object[] {new[] {"a", "b", "c", "C"}}
+            new SimpleObjectCollection("x", "y", "z", "Z"),
+            new[] {'A', 'B', 'C', 'c'},
+            new[] {"a", "b", "c", "C"}
```

Comment:
Why is the nature of the test changing here? The original outer array was of objects, which is the standard type used for argument lists in .NET. The argument itself is an IEnumerable, which is satisfied by each of the inner collections given. The new reformulation is simpler to write for sure, but the old one should still work. From your comment, I gather it does not.